### PR TITLE
Onboard codait-advo team

### PR DESCRIPTION
**Pull request by 4n4nd**
Resolves https://github.com/operate-first/support/issues/202

### CODE REVIEW

File: cluster-scope/base/namespaces/codait-advo/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: codait-advo
```
Comment (harshad16) [inline cluster-scope/base/namespaces/codait-advo/kustomization.yaml:5]: is this to be added to zero cluster?
missing reference here: https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/moc/zero/kustomization.yaml

**Review by harshad16 — COMMENTED** [review]
/lgtm

just saw that namespace was not added to zero cluster kustomize file.

File: cluster-scope/base/namespaces/codait-advo/kustomization.yaml (hunk 1 above)
Comment (4n4nd) [reply cluster-scope/base/namespaces/codait-advo/kustomization.yaml:5]: good catch :+1:

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**